projects/telemetry/src/nmea_checksum.c

```diff
@@ -0,0 +1,82 @@
+#include "nmea_checksum.h"
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+static const int HEX_ERROR = 16;
```

Comment:
Use a #define instead of a variable. Also, make sure you're following the naming scheme.